cupboard_app/test_queries.py

```diff
@@ -503,13 +582,23 @@ def test_create_list_ingredient(self):
         """
         Testing create_list_ingredient creates an ingredient dictionary
         """
+
+        create_custom_ingredient(self.user1.username, self.ing2.name, self.ing2.type)
```

Comment:
Use the django query instead to create the custom ingredient because we are not testing the function (i.e. with this we are dependent on if the create_custom_ingredient function works)